app/controllers/dashboard_controller.rb

```diff
@@ -0,0 +1,4 @@
+class DashboardController < ApplicationController
+  before_action :authenticate_user!
```

Comment:
I will authenticate user only on Dashboard?

Reply:
In the current "version" of the app (all features present in this branch) dashboard is the only page where authorization is needed. We plan to use both sessions and cancancan to restrict unauthorized users from using admin functions later.